src/notte/sdk/endpoints/persona.py

```diff
@@ -0,0 +1,169 @@
+from collections.abc import Sequence
+from typing import Unpack
+
+from pydantic import BaseModel
+from typing_extensions import final, override
+
+from notte.sdk.endpoints.base import BaseClient, NotteEndpoint
+from notte.sdk.types import (
+    EmailResponse,
+    EmailsReadRequest,
+    EmailsReadRequestDict,
+    PersonaCreateRequest,
+    PersonaCreateRequestDict,
+    PersonaCreateResponse,
+    SMSReadRequest,
+    SMSReadRequestDict,
+    SMSResponse,
+    VirtualNumberRequest,
+    VirtualNumberRequestDict,
+    VirtualNumberResponse,
+)
+
+
+@final
+class PersonaClient(BaseClient):
+    """
+    Client for the Notte API.
+
+    Note: this client is only able to handle one session at a time.
+    If you need to handle multiple sessions, you need to create a new client for each session.
+    """
+
+    # Session
+    EMAILS_READ = "{persona_id}/email/read"
+    SMS_READ = "{persona_id}/sms/read"
+    CREATE_NUMBER = "{persona_id}/create-number"
+    CREATE_PERSONA = "create"
+
+    def __init__(
+        self,
+        api_key: str | None = None,
+    ):
+        """
+        Initialize a PersonaClient instance.
+
+        Initializes the client with an optional API key for persona management.
+        """
+        super().__init__(base_endpoint_path="persona", api_key=api_key)
```

Comment:
would set it to `personas` similarly to other endpoints (`sessions`, `agents`, etc.)